cmake/projects/cub/hunter.cmake

```diff
@@ -21,6 +21,17 @@ hunter_add_version(
     c90ecb5fd3c393371abb9cfbf61d4fd5e36564f1
     )
   
+ hunter_add_version(
+    PACKAGE_NAME
+    cub
+    VERSION
+    1.8.0-p0
+    URL
+    "https://github.com/a-sevin/cub/archive/v1.8.0-p0.tar.gz"
```

Comment:
We already have a fork here:
- https://github.com/hunter-packages/cub

I've created branch `hunter-1.8.0` based on `v1.8.0` tag:
- https://github.com/hunter-packages/cub/tree/hunter-1.8.0

Please send patches there first.